### ISSUE TITLE
LevT on new dataset

### ISSUE THREAD

**Issue by zylm** [facebookresearch/fairseq#4024]
### Before asking:
1. search the issues.
2. search the docs.

#### What is your question?
In LevT, how do you prevent the generated insertion numbers from becoming all zeros?

Train LevT on my data set, the training loss will become nan. Looking at the reason, it is found that the predictions of insert number will all become 0. Have you ever encountered this situation? Is there any way to avoid or alleviate this problem?

#### What's your environment?

 - fairseq Version (e.g., 1.0 or main):
 - PyTorch Version (e.g., 1.0)
 - OS (e.g., Linux):
 - How you installed fairseq ( source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:
